### PR TITLE
Initial Implementation of ISO 20022 Records Library for Financial Messaging

### PR DESCRIPTION
**Pull request by Ajai-Suvendran**
## Purpose
The purpose of this pull request is to introduce the ISO 20022 Records Library , which provides comprehensive support for handling ISO 20022 financial messaging records. 

## Goals
This library enables developers to work with structured data models generated from the ISO 20022 XSDs, facilitating easier creation, manipulation, and parsing of ISO 20022-compliant messages, offering a standardized approach for handling complex financial data within Ballerina applications.

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[package]
+org = "ballerinax"
+name = "iso20022records"
+version = "1.0.0"
+distribution = "2201.10.1"
+
+
```
Comment (SasinduDilshara) [inline ballerina/Ballerina.toml]:
```suggestion
```

Comment (SasinduDilshara) [reply ballerina/Ballerina.toml]: Lets add all necessary lines at the end of each files

File: ballerina/camt.026.001.10.bal [hunk 2]
```diff
@@ -0,0 +1,52 @@
+public type Camt026Document1 Camt026Document;
```
Comment (Ashi1993) [inline ballerina/camt.026.001.10.bal:1]: Shall we add the link refering to the type for each

File: ballerina/camt.034.001.07.bal [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    SkipPayload Any;
+|};
+
+public type ProprietaryData7 record {|
+    Max35Text Tp;
+    ProprietaryData6 Data;
+|};
+
+public type SkipPayload record {|
+|};
```
Comment (Ashi1993) [inline ballerina/camt.034.001.07.bal]: Add EoF line, Check all files

**Review by Ashi1993 — APPROVED** [review]

**Review by gimantha — APPROVED** [review]

File: ballerina/Ballerina.toml (hunk 1 above)
Comment (Ajai-Suvendran) [reply ballerina/Ballerina.toml]: fixed in `f057766`

File: ballerina/camt.034.001.07.bal (hunk 3 above)
Comment (Ajai-Suvendran) [reply ballerina/camt.034.001.07.bal]: fixed in `f057766`

**Pull request merged**